framework/src/main/java/org/tron/core/config/args/Args.java

```diff
@@ -1545,40 +1544,14 @@ private static void bindIp(final com.typesafe.config.Config config) {
     }
   }
 
-  private static void externalIp(final com.typesafe.config.Config config) {
+  private static void externalIp(final com.typesafe.config.Config config, P2pConfig p2pConfig) {
```

Comment:
I think the second parameter is redundant. Can it be removed?

Reply:
@chengtx01  Yes, i have already removed it.